packages/app-store/routing-forms/lib/transformResponse.ts

```diff
@@ -8,5 +8,20 @@ export default function transformResponse({
   value: Response[string]["value"];
 }) {
   // type="number" still gives value as a string but we need to store that as number so that number operators can work.
-  return field.type === "number" && typeof value === "string" ? Number(value) : value;
+  if (field.type === "number") {
+    if (typeof value === "string") {
+      return Number(value);
+    }
+    return value;
+  }
+  if (field.type === "multiselect") {
+    if (value instanceof Array) {
+      return value;
+    }
```

Comment:
Why do we have this condition if both cases are returning the value? 🤔

Reply:
It's easier to read for me. Like a switch statement there is a check for type of field and then you handle it's cases.